tests/unittests/test_tabular.py

```diff
@@ -164,7 +164,7 @@ def run_tabular_benchmark_toy(fit_args):
     predictor = task.fit(train_data=train_data, label=dataset['label_column'], output_directory=savedir, **fit_args)
     try:
```

Comment:
Can we add a call to `print(predictor.feature_metadata.X)` where X = each of the attributes in this object?
Just to make sure future changes to the object don't break users code

Reply:
added